src/main.js

```diff
@@ -159,18 +168,36 @@ function backHome() {
   mainPosterView.classList.remove('hidden');
   savedPostersView.classList.add('hidden');
   posterFormView.classList.add('hidden');
-
 }
 
+
 function showFormPage() {
   // console.log('Hello');
   posterFormView.classList.remove('hidden');
   mainPosterView.classList.add('hidden');
   // savedPostersView.classList.add('hidden');
+}
 
 
+function userInput(event) {
+  event.preventDefault();
+  console.log('hey there');
+  var imageUser = userImageUrl.value;
+  var titleUser = userTitle.value;
+  var quoteUser = userQuote.value;
+  currentPoster = new Poster(imageUser, titleUser, quoteUser);
+  //form.reset();
+  backHome();
+  displayPoster(currentPoster);
+  pushUserInput(imageUser, titleUser, quoteUser)
+}
 
+function pushUserInput(userImageUrl, userTitle, userQuote) {
```

Comment:
Let's also remind ourselves to go back and make sure all of our spacing is correct between functions and what-not.